STM32/Util/Src/systeminfo.c

```diff
@@ -26,6 +26,8 @@ static struct BS
     float underVoltage;
     float overVoltage;
     float overCurrent;
+    BoardType boardType;
+    pcbVersion pcbVersion;
 } BS = {0, 0, 0, 0};
```

Comment:
Does the initialization (line 31) need default values for the added BoardType and pcbVersion?